### PR TITLE
Created initial template for testcases

### PR DESCRIPTION
**Pull request by VKelvinWang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,28 @@ $ cp .env.example .env
...
+
+3. Run ubuntu on your Windows computer. Afterwards, it will ask you to create initial username and password.
+
+4. Run the following command after you registered into Ubuntu
+
+```
+$ sudo apt update
+```
+
+5. start your computer to allow these installs to take effect.
```
Comment (vck3000) [inline README.md:66]: Let's not specify the steps to install WSL. These can change at any time and we're better off directing future readers to the official WSL docs :). 

Could you please add a note somewhere suitable that Windows isn't natively supported, and to recommend them to install and use WSL?

File: src/gameplayv2/resistance/tests/component.test.js
Comment (vck3000) [inline src/gameplayv2/resistance/tests/component.test.js:1]: Let's make everything typescript files (.ts) given we have the chance to start with types :).

File: src/gameplayv2/resistance/tests/component.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    //or for an array:
+    const arraychar = ['a', 'b']
+    const array = [1, 2]
+    //this will throw an Error
+    expect(cloneArray(array)).toEqual(arraychar);
+});
+
+//take a look at rewards/tests/getRewards.test.ts
+//if you want to run this, do npm test on terminal
```
Comment (vck3000) [inline src/gameplayv2/resistance/tests/component.test.js:20]: Nit: Usually we have a space after the `//`. Prettier (auto-formatter) should handle this for you! I'd recommend installing it if you haven't already!

File: src/gameplayv2/resistance/tests/component.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Resistance } from '../entities/resistance'
+
+test('properly initialize the Spy entity', () =>{
+    //do test case here..
+});
+
+test('properly initialize the Resistance entity', () =>{
```
Comment (vck3000) [inline src/gameplayv2/resistance/tests/component.test.js:7]: `() =>{` -> `() => {` (prettier auto-formatter!

File: README.md (hunk 1 above)
Comment (VKelvinWang) [reply README.md:66]: Will add a note into that will specify Windows is not natively supported :)

File: README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Other useful resources:
 - [Visual Studio Code](https://code.visualstudio.com/), a code editor/IDE by Microsoft.
 - [nvm](https://github.com/nvm-sh/nvm), a useful tool to manage your node versions.
 
-### Installing
+### Installing on a local environment
```
[the remark this answers is not in the document]
Comment (VKelvinWang) [reply README.md:21]: I'll chane it to installation.

**Pull request closed without merging**